libs/cypress-indexeddb/src/lib/cypress-indexeddb.ts

```diff
@@ -0,0 +1,167 @@
+const DATABASES = new Map<string, IDBDatabase>();
+const STORES = new Map<string, IDBObjectStore>();
+
+export function setupIDBHelpers(): void {
```

Comment:
maybe we could automatically execute this function when the module is loaded for the first time?
This way we can limit the setup necessary to just importing this file.

```
// this would be all the setup necessary
import '@this-dot/cypress-indexeddb'
```

```suggestion
setupIDBHelpers()

export function setupIDBHelpers(): void {
```

Reply:
Great tip, but I rehauled the whole thing since so it will be not applied via suggestions.